ansible/roles/rrdstats/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+rrdstats_script_path: /opt/pydis/generate-rrdtool-stats.sh
+rrdstats_env_path: /etc/opt/pydis-rrdstats.env
+
+rrdstats_pg_username: rrdtool
```

Comment:
all pg users have been put in the postgres role, might as well do the same here ?